content/concepts/issued-currencies/rippling.md

```diff
@@ -52,7 +53,9 @@ The NoRipple flag makes certain paths invalid, so that they cannot be used to ma
 
 ## The DefaultRipple Flag
 
-The DefaultRipple flag is an account setting that enables rippling on all trust lines by default. Gateways and other currency issuers MUST enable this flag for other addresses to be able to send those the currencies among themselves.
+The DefaultRipple flag is an account setting that enables rippling on all incoming trust lines by default. Gateways and other currency issuers MUST enable this flag for other addresses to be able to send those the currencies among themselves.
```

Comment:
Nit: What does "those" refer to? Is there a missing or extra word there?

>to be able to send *those* the currencies among themselves.

Reply:
Yeah, I think that's an old typo. Should be either "those currencies" or "the currencies" not "those the currencies"

Reply:
Good eyes, @jhaaaa.  I missed that when I read it over.